base-libs/cracklib/autobuild/beyond

```diff
@@ -1,14 +1,43 @@
-./util/cracklib-format cracklib-dicts/* | \
-./util/cracklib-packer "$PKGDIR"/usr/share/cracklib/pw_dict
-./util/cracklib-format "$PKGDIR"/usr/share/cracklib/cracklib-small | \
-./util/cracklib-packer "$PKGDIR"/usr/share/cracklib/cracklib-small
-rm -f "$PKGDIR"/usr/share/cracklib/cracklib-small
+abinfo "Preparing to build Python bindings separately ..."
+# Adpated from Debian
+PYBLDDIR="$SRCDIR"/abpybuild
+mkdir -pv "$PYBLDDIR"
+cd "$PYBLDDIR"
 
+abinfo "Configuring Python binding source tree ..."
+"$SRCDIR"/configure $AUTOTOOLS_DEF ${AUTOTOOLS_AFTER/--without-python/}
```

Comment:
Better make `--enable-python` explicit? Why even bother with this hack in the first place, wouldn't cracklib build the Python 3 bindings if you specified `--with-python` and `PYTHON=/usr/bin/python3` in `AUTOTOOLS_AFTER=`?

Reply:
Well, yes, it wouldn't. Cracklib's current `configure` cannot "see" Python 3 headers, and I have no idea to fix it. And even Python bindings originally built with Python 2 are incomplete, because according to cracklib package file lists from other distros, cracklib should have had libraries depended on `libpython<VER>.so`, but ours doesn't.  And Fedora and Debian build these bindings separately, so I follow their methods.